src/app.ts

```diff
@@ -165,19 +195,25 @@ export default (
       method: 'updateSecret',
     });
 
+    // I think for now we can use the taskName Prefix and edit it when we
+    // are adding additional information to the check
+    const checkResponse = await context.octokit.checks.create({
+      owner: await getOwner(context),
```

Comment:
This can be consumed directly from the context

```suggestion
      owner: context.payload.installation.account.login
```

Reply:
Resolved in a pushed commit.